webapp/lib/charts.dart

```diff
@@ -4,6 +4,19 @@ import 'dart:html';
 import 'package:crypto/crypto.dart';
 import 'package:chartjs/chartjs.dart' as chartjs;
 
+
+init() {
+  chartjs.ChartTooltipPositioner custom = (List<dynamic> elements, chartjs.Point eventPosition) {
+    if (elements.isEmpty) {
+      return null;
+    }
+    return new chartjs.Point(x: eventPosition.x, y: eventPosition.y);
+  };
+  var tooltipConfig = chartjs.ChartTooltipsStaticConfiguration()
+    ..positioners = {'custom': custom};
+  chartjs.Chart.Tooltip = tooltipConfig;
```

Comment:
ok, found the magic:
```suggestion
  chartjs.Chart.Tooltip.positioners.custom = custom;
```